gemrb/core/Map.h

```diff
@@ -458,6 +460,24 @@ class GEM_EXPORT Map : public Scriptable {
 	MainAmbients dayAmbients;
 	MainAmbients nightAmbients;
 
+	enum FogDirection : uint8_t {
+		N = 1,
+		W = 2,
+		NW = N|W, // 3
+		S = 4,
+		SW = S|W, // 6
+		E = 8,
+		NE = N|E, // 9
+		SE = S|E // 12
+	};
+
+	// Size of Fog-Of-War shadow tile (and bitmap)
+	static constexpr int CELL_SIZE = 32;
+
+	// Buffers of const allocation size
+	static std::vector<float> fogVertices;
+	static std::vector<Color> fogColors;
+
```

Comment:
These were all moved to the renderer, so can be removed here and in the cpp (the two vectors).

Reply:
Oh yeah, I missed cleaning up that section with the lines above as well.